spec/controllers/appointments_controller_spec.rb

```diff
@@ -63,6 +63,46 @@
       end
     end
 
+    describe "filtering by days overdue" do
+      it "displays patients only less than one year overdue when checked" do
+        really_overdue_appointment = create(:appointment,
+          facility: facility_2,
+          scheduled_date: 380.days.ago,
+          status: 'scheduled'
+        )
+        create(:blood_pressure, patient: really_overdue_appointment.patient, facility: facility_2)
+        really_overdue_patient_ids = really_overdue_appointment.patient_id
+
+        get :index, params: {
+          search_filters: ["only_less_than_year_overdue"],
+          per_page: 'All'
+        }
+
+        patient_ids = (overdue_appointments_in_facility_1 + overdue_appointments_in_facility_2).map(&:patient_id)
+        expect(assigns(:patient_summaries).map(&:id)).to match_array(patient_ids)
+        expect(assigns(:patient_summaries).map(&:id)).to_not match_array(really_overdue_patient_ids)
```

Comment:
Rather than matching an array, can we assert `.to_not include(really_overdue_patient_id)`? The `match_array` will pass even if the ID _is_ in the array, but doesn't exactly match, right?

Reply:
Fixed